tests/manage/z_cluster/nodes/test_non_ocs_taint_and_toleration.py

```diff
@@ -81,12 +81,18 @@ def test_non_ocs_taint_and_tolerations(self):
         )
 
         # Taint all nodes with non-ocs taint
+        logger.info("Taint all nodes with non-ocs taint")
         ocs_nodes = get_worker_nodes()
         taint_nodes(nodes=ocs_nodes, taint_label="xyz=true:NoSchedule")
 
+        resource_name = constants.DEFAULT_CLUSTERNAME
+        if config.DEPLOYMENT["external_mode"]:
+            resource_name = constants.DEFAULT_CLUSTERNAME_EXTERNAL_MODE
+
         # Add tolerations to the storagecluster
+        logger.info("Add tolerations to storagecluster")
```

Comment:
why are comment sessions added as log messages ? i see this change throughout the test

Reply:
Bcz logs get printed during run time and helps debug if the test fails

Reply:
Anybody who looks into the test directly or in this even the test logs understands the test execution with the current implementation. Hence lets remove the comment lines so we dont have to repeat the same statements in comment and in log messages.

Reply:
sure, will remove the existing comments

Reply:
Done